Docs/Source/systems/architecture.md

```diff
@@ -10,3 +10,4 @@ ClientSim uses the observer pattern to send events within the system that anythi
 
 ClientSim’s architecture uses a manually-handled dependency injection. On creation of a system, all dependencies are passed to it, either through its constructor or through an initialization method. Dependencies are structured as providers, and must extend an interface that declares what methods it provides. When a class needs a specific item, it depends on the provider interface instead of the class that implements it. This allows for different implementations of the provider without the dependent code needing to change. The provider pattern allows for dependencies to easily be mocked in tests.
 
+![test](../images/editor-screenshot.png)
```

Comment:
This threw the error `Error: Image docs/images/editor-screenshot.png used in docs/systems/architecture.md not found.`

Changing it to start with `/images/` like said in documentation fixed it
```suggestion
![test](/images/editor-screenshot.png)
```

Reply:
Thanks for the fix, but this image should actually be removed, it doesn't belong here 😛